workshops/views.py

```diff
@@ -2185,8 +2198,11 @@ def export_members(request):
 @admin_required
 def workshops_over_time(request):
     '''Export JSON of count of workshops vs. time.'''
+    endpoint = '{}?{}'.format(reverse('api:reports-workshops-over-time'),
```

Comment:
OK, this answers the question above. But isn't there something in the library to construct the URL with query parameters, rather than us using string formatting?